src/3_PB/PianoDiProgetto/sections/ConsuntivoSprint/NonoSprint.typ

```diff
@@ -0,0 +1,20 @@
+#import "../../const.typ": Re_cost, Am_cost, An_cost, Ve_cost, Pr_cost, Pt_cost
+#import "../../functions.typ": rendicontazioneOreAPosteriori, rendicontazioneCostiAPosteriori, glossary
+
+
+== Nono #glossary[sprint]
+
+*Inizio*: Venerdì 16/01/2024
+
+*Fine*: Giovedì 22/01/2024 
+
+#rendicontazioneOreAPosteriori(sprintNumber: "09")
+
+#rendicontazioneCostiAPosteriori(sprintNumber: "09")
+
+=== Analisi a posteriori
+La retrospettiva ha evidenziato come il preventivo del totale delle ore sia stato rispettato. Tuttavia possiamo notare alcune differenze nell'ammontare delle ore di alcuni ruoli. In particolare, il ruolo del Verificatore è quello che risulta più differente rispetto al preventivo, con una totale delle ore inferiore. Questo non ci risulta di particolare preoccupazione. Infatti, si motiva con il fatto che le modifiche e integrazioni apportate ai documenti sono state minori rispetto a #glossary[sprint] passati e avendo due Verificatori la verifica è stata piuttosto rapida. Notiamo invece l'accuratezza del preventivo per il ruolo di Progettista, che nonostante venga utilizzato per la prima volta, con l'inizio della stesura delle _Specifiche Tecniche v1.0_, risulta esser stato stimato con precisione, indice di una maturata consapevolezza nella pianificazione delle attività. Con un leggero sforamento le ore da Analista preventivate hanno permesso di rispettare la pianificazione e di completare la correzione dell'_Analisi dei Requisiti v1.0_ secondo le correzioni ricevute in revisione #glossary[RTB].
+Una strategia efficace per inibire il rischio RC1 è stata la scelta di svolgere gli #glossary("Stand-up meeting") remoti con una frequenza di due volte a settimana, che in unione al meeting interno settimanale, ha permesso di tenere sotto controllo l'avanzamento delle attività e di discutere tempestivamente eventuali problematiche, risultando un impegno sostenibile per il gruppo e quindi una pratica da mantenere anche in futuro.
+
+==== Aggiornamento della pianificazione e gestione dei rischi
+Il gruppo ha preso coscienza del fatto che il preventivo per questo #glossary[sprint] considerava fin troppe poche ore rispetto a quelle che si potevano impiegare a livello di risorse e che sono necessarie a completare le attività pianificate per la revisione #glossary[PB]\; di conseguenza, in ottica di arginare il rischio RP3, vengono aggiornati i preventivi successivi in modo da aumentare le ore di lavoro. In particolare, poiché questi primi #glossary[sprint] della #glossary[PB] richiedono principalmente attività di progettazione, si prevede un aumento delle ore per il ruolo di Progettista, ruolo che sarà ricoperto da un maggior numero di membri del gruppo.
```

Comment:
molto bene, unica cosa non ci sono citati per niente i risultati delle metriche del pdq, ma volendo per sto sprint puo rimanere anche così dato che non ricordo se precedentemente erano state citato o meno. 
Più che altro dovremmo tenerle in considerazione nei prossimi forse.

Reply:
Per questo ho bisogno di un po' di tempo per capire se e come inserirle.

Reply:
okay ne abbiamo parlato apposto